style.css

```diff
@@ -454,3 +467,255 @@ html {
 .footer-social-item {
   color: #505f79;
 }
+
+.header-overly {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
```

Comment:
- [ ] Good job on making both the desktop and mobile versions look great 👏 However there are still some issues with some screen sizes, the website is not fully responsive yet, kindly check the gif below and try to fix the issues 👍 

![Animation](https://user-images.githubusercontent.com/50721479/215275160-15b13fde-e195-4807-a4fd-7350d35f24c6.gif)